other/bootstrap_daemon/tox_bootstrap.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Tox DHT Bootstrap Daemon
+After=network.target
+
+[Service]
+Type=forking
+PermissionsStartOnly=true
+ExecStartPre=/usr/bin/touch /var/run/tox-bootstrapd.pid
```

Comment:
If PID is used and specified - systemd will delete it on service stop. Everything is fine.
if PID is not used and not specified - there is nothing to do prior start. 

But now PID is used and mandatory, right? So we need to ensure the deletion of PID after stop in systemd, and creation with correct privileges before start.